FNML/ontology.md

```diff
@@ -96,14 +98,14 @@ The mapping challenge [Join on literals](https://github.com/kg-construct/mapping
 
 When this triples map **does not specify a subject map**, a blank node should be generated for the subject.
 
-<div class="note">
-We should probably define a new resource that is a subclass of rr:TriplesMap,
-as the description above is disjoint with rr:TriplesMap, as logical source and subjectMap are not optional in [[R2RML]]
-</div>
+### fnml:functionValue
+
+<dfn>fnml:functionValue</dfn> connects the knowledge graph generating triples map using a [=function term map=] with a [=function triples map=].
```

Comment:
Should we use the term knowledge graph in the spec? I would prefer RDF dataset. If we do decide to use knowledge graph, we should define it.

Reply:
used RDF dataset instead of knowledge graph throughout the dataset, indeed much better